src/main/java/org/opensearch/testcontainers/OpensearchContainer.java

```diff
@@ -39,6 +46,8 @@ public class OpensearchContainer<SELF extends OpensearchContainer<SELF>> extends
     // HTTPs,
     // along with Basic Auth being used.
     private boolean disableSecurity = true;
+    private boolean requireInitialPassword = false;
+    private String password = DEFAULT_PASSWORD;
```

Comment:
Should this not be initialized here?

Reply:
It simplifies a bit the logic, for OS < 2.12